src/qcodes/instrument_drivers/oxford/triton.py

```diff
@@ -461,7 +461,7 @@ def _get_temp_channel_names(self, file: str) -> None:
             options = config.options(section)
             namestr = '"m_lpszname"'
             if namestr in options:
-                chan_number = int(section.split("\\")[-1].split("[")[-1]) + 1
+                chan_number = int(re.findall('\d+', section)[-1]) + 1
```

Comment:
```suggestion
                chan_number = int(re.findall(r'\d+', section)[-1]) + 1
```
`\` that is not for explicit control seqs like `\n` needs to either be escaped or be in a raw string. IHMO raw strings are probably easier to read so thats what I propose above